index.d.ts

```diff
@@ -31,6 +31,22 @@ declare namespace prettyBytes {
 		```
 		*/
 		readonly bits?: boolean;
+		
+		/**
+		Format the number using the [Binary Prefix](https://en.wikipedia.org/wiki/Binary_prefix) instead of the [SI Prefix](https://en.wikipedia.org/wiki/SI_Prefix). This can be useful as memory is much better represented by powers of 2.
```

Comment:
Can you linkify `memory is much better represented by powers of 2` to a source on why it's better?

Reply:
The text should also clearly point out that it's not appropriate to use this for file sizes.

Reply:
I think it might explain why in the Binary prefix wikipedia page already linked. 

> By the mid-1960s, binary addressing had become the standard architecture in most computer designs, and main memory sizes were most commonly powers of two. This is the most natural configuration for memory, as all combinations of their address lines map to a valid address, allowing easy aggregation into a larger block of memory with contiguous addresses.

I didn't write the statement in the comment so I don't have a problem removing it either if you feel like the language is too strong. 

Also wanted to note, that why file sizes are denoted in base10 in Linux and macOS, that in Windows they are represented by base2. But also sometimes a developer might choose a different method than the other preferred method for the operating system.

Reply:
I have updated the comment.

Reply:
> Also wanted to note, that while file sizes are denoted in base10 in Linux and macOS, that in Windows they are represented by base2.

Windows would have switched to Base10 too if they weren't on the extreme end of backward compatibility.

Reply:
Ok